sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1880,7 +1880,7 @@ namespace Azure { namespace Core {
 
           if (!IsCrlValid(crl))
           {
-            Log::Write(Logger::Level::Informational, "Discarding outdated CRL");
+            Log::Write(Logger::Level::Error, "Discarding outdated CRL");
             X509_CRL_free(*it);
             *it = nullptr;
             continue;
```

Comment:
This one seems potentially recoverable error, since it is in a loop, should it be Warning? If it is not recoverable, we'll probably print an Error later, and customers who investigate would increase log level to Warning+?

Reply:
It could be recoverable. But it is a critical diagnostic when trying to debug why CRL validation failed.

These will all be reduced to a lower level in a future PR. But this PR is all about making the errors consistent so that they can be identified later.